presto-docs/src/main/sphinx/develop/worker-protocol.rst

```diff
@@ -73,6 +77,10 @@ response includes:
   request the next chunk as ``X-Presto-Page-End-Sequence-Id`` HTTP header,
 * An indication that there are no more results as ``X-Presto-Buffer-Complete``
   HTTP header with the value of "true".
+* The remaining buffered bytes in the output buffer as ``X-Presto-Buffer-Remaining-Bytes``
```

Comment:
> This should return a comma separated list of the size in bytes of
  the pages that can be returned in the next request
  
From what I understand the current implementation returns a single number containing the total size. This is inconsistent with what Prestissimo returns.

Data streams fetched from coordinator are relatively small. Though I'm concerned about future incompatibilities. For example the assumptions in the algorithm in Prestissimo ExchangeClient may change and it may stop working as intended. How difficult would it be to make it consistent? (return an actual list)

Reply:
Not hard, just more code.  I'll follow up with another PR.

Reply:
#22060 demonstrates how this looks like, PTAL and let me know if you prefer that one.